src/connection_pool.rs

```diff
@@ -90,7 +101,7 @@ where
 
         let barrier = Arc::new(Barrier::new(2));
 
-        let mut resolver = cpo.resolver;
+        // let mut resolver = cpo.resolver;
```

Comment:
Should this be removed since it's commented?

Reply:
Good eye. Will get rid of that.